nixos/modules/services/networking/wireguard.nix

```
@@ -170,10 +170,22 @@ let
 
   # peer options
 
-  peerOpts = {
+  peerOpts = self: {
 
     options = {
 
+      name = mkOption {
+        default =
+          replaceChars
```

I can't really comment on the change itself, but I know that `replaceChars` is deprecated.
Please use `replaceStrings` instead :)

https://github.com/NixOS/nixpkgs/blob/37878b459ddfbde32fafa1fabd2bd78f5b26e02c/lib/strings.nix#L486-L487


```suggestion
      name = mkOption {
        default =
          replaceStrings
```

---

Thanks. `replaceChars` was copypasted from 22.11, where this patch was developed against. Haven't seen that in master it's already `replaceStrings`.